src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.js

```diff
@@ -45,22 +37,25 @@ const BaseAnchorForCommentsOnly = ({
                                 showContextMenu(
                                     CONTEXT_MENU_TYPES.LINK,
                                     event,
-                                    href,
+                                    props.href,
                                     lodashGet(linkRef, 'current'),
                                 );
                             }
                         }
                 >
                     <Text
                         ref={el => linkRef = el}
-                        style={StyleSheet.flatten(style)}
+                        style={StyleSheet.flatten(props.style)}
```

Comment:
Like maybe in the `Text` component here we pass `style` and then overwrite it by spreading `props` below - but before it was fine because of rest syntax?

Reply:
Should be fixed